dev-haskell/primitive-extras/primitive-extras-0.10.2.1-r1.ebuild

```diff
@@ -20,7 +20,7 @@ RDEPEND=">=dev-haskell/cereal-0.5.5:=[profile?] <dev-haskell/cereal-0.6:=[profil
 	>=dev-haskell/focus-1:=[profile?] <dev-haskell/focus-1.1:=[profile?]
 	>=dev-haskell/foldl-1:=[profile?] <dev-haskell/foldl-2:=[profile?]
 	>=dev-haskell/list-t-1.0.1:=[profile?] <dev-haskell/list-t-1.1:=[profile?]
-	>=dev-haskell/primitive-0.7:=[profile?] <dev-haskell/primitive-0.10:=[profile?]
+	>=dev-haskell/primitive-0.7:=[profile?]
```

Comment:
Another removal of an upper bound with no change to `CABAL_HACKAGE_REVISION` or `CABAL_CHDEPS`.